src/managers/contact_manager.py

```diff
@@ -134,3 +134,24 @@ def search_by_phone_number(self, phone_number: str) -> List[Contact]:
                     matching_contacts.append(contact)
 
         return matching_contacts
+    
+    def handle_command(self, command: str) -> None:
+        """
+        Handler different commands. Responds to "all" command by displaying all contacts.
+        
+        """
+        if command.lowe() == "all":
+            self.display_all_contacts()
+
+    def display_all_contacts(self) -> None:
+        """
+        Displays all contacts in readable format. Shows a message if no contacts.
+
+        """
+        if not self.contacts:
+            print("No contacts available.")
+        else:
+            print("All Contacts:")
+            for contact in self.contacts:
+                print (f"ID: {contact.id}, Name: {contact.name}, Address: {contact.address},"
```

Comment:
I recommend simplifying this logic:
1. Create the __str__ method in the Contact class and generate the output of all contacts there, this will improve the readability of the code and ensure that all Contact objects are generated in the same way.
2. The display_all_contacts method already displays all Contacts, for example (from homework):
`contact_list = "\n".join(str(record) for record in book.values())
    return f"Contacts:\n{contact_list}"`